adapt-and-extend/theming/customize-barceloneta.rst

```diff
@@ -0,0 +1,181 @@
+
+Note: While Plone supports “less” syntax in style files, you can of course also write plain css only.
+
+These steps show you how to register your own styles file so that it gets included into the Plone theme.
+
+code here
+
+Note: If you are changing styles in your style file, remember to recompile (describe the actual steps necessary from changing a style in the file to actually seeing the difference in your browser).
+
+
+Change CSS that’s already there
++++++++++++++++++++++++++++++++
+
+The previous steps should already have gotten your site into a quite decent state. You can do already a lot with custom css, but as soon as your custom css file get’s quite long, things become crowded and you might get a desire for more structure. Instead of now adding even more of your own custom css files and slowly duplicating most of the css, you can actually also exchange existing Plone css.
+
+Plone keeps its css in a very well structured, componentized way. If you want to change the styling for a certain area or component, chances are good that you will find them already grouped in a dedicated less file. 
+
+Take a look what’s available here… (where are the barceloneta less files?)
+
+If you can identify the styles group you want to exchange, perform the following steps to unregister the original one and register your own.
+
+
+Customise Special Elements
+--------------------------
+
+Change the toolbar logo
++++++++++++++++++++++++
+
+By now you have probably noticed that there are a few more elements that you might want to alter. One is the little Plone logo in the toolbar. If you want to change that, you can do that by …
+
+code here
+
+ 
+The Omnipresent Icons
++++++++++++++++++++++
+
+Plone doesn’t use images as icons for content types or actions anymore. Instead it uses a fontello font which is configured to contain all the icons needed as characters. If you want to change these, yo can of course provide your own and override via css, but you can also upload the font definition file to fontello, reconfigure it there and readd the modified font file to your customisation.
+
+Here is how you upload and modify the fontello font file:
+
+code here
+
+Here is how you override the font in your own code:
+
+code here
+
+Change the markup (jbot)
+++++++++++++++++++++++++
+
+So far we have only touched the styles and resources. And while you can do a lot with css, there are moments where you might need an insane amount of effort or simply unmaintainable hacks to achieve what you want. In the contrary, just changing e.g. the order  of the markup might just do what you need.
+
+Luckily, changing components of the Plone theme is easy and straightforward. Once you have registered jbot for your package (link to jbot installation), you can copy the original template file and place it into your package - just following a simple naming convention:
```

Comment:
we should try to avoid to use here the word easy and maybe even straightforward, this is confusing for new people with less skills